src/main/java/com/bunq/sdk/http/ApiClient.java

```diff
@@ -85,15 +85,15 @@
   /**
    * Header value to disable the cache control.
    */
-  private static final String CACHE_CONTROL_NONE = "no-cache";
+  public static final String CACHE_CONTROL_NONE = "no-cache";
 
   /**
    * Prefix for bunq's own headers.
    */
   private static final String USER_AGENT_BUNQ = "bunq-sdk-java/0.13.0";
-  private static final String LANGUAGE_EN_US = "en_US";
-  private static final String REGION_NL_NL = "nl_NL";
-  private static final String GEOLOCATION_ZERO = "0 0 0 0 000";
+  public static final String LANGUAGE_EN_US = "en_US";
```

Comment:
public under private?

Reply:
hmm., i normally group constants in groups, not on visibility. Error constants on top, ten the first used group in the class. So there could indeed be a mixture of visibility in a specific group.